hack/make/.binary

```diff
@@ -49,7 +49,7 @@ hash_files() {
 				;;
 			linux/arm)
 				case "${GOARM}" in
-					5 | "")
+					5 | 6 | "")
```

Comment:
@lmbarros minimal change required to build `linux/arm/v6` (I have no idea how it was previously building without this).